lightspeed/modules/proc_create-lightspeed-instance.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: PROCEDURE
+
+[id="create-lightspeed-instance_{context}"]
+
+= Creating and deploying a {LightspeedShortName} instance
+
+Use this procedure to create and deploy a {LightspeedShortName} instance to your namespace.  
+
+.Prerequisites
+* You have created connection secrets for both {PlatformName} and {ibmwatsonxcodeassistant}.
+
+.Procedure
+
+. Go to {OCP}.
+. Select *Operators → Installed Operators*.
+. From the *Projects* list, select the namespace where you want to install the {LightspeedShortName} instance and where you created the connection secrets.
+. Locate and select the *{PlatformNameShort} (provided by Red Hat)* operator that you installed earlier.
+. Select *All instances → Create new*.
+. From the *Create new* list, select the *Ansible Lightspeed* modal.
+. Ensure that *Form view* is selected as the configuration mode. 
```

Comment:
To avoid passive voice, I would rewrite as "Ensure that you have selected Form View as the config mode"